openapi/views/openapi_view.xml

```diff
@@ -0,0 +1,148 @@
+                                <field name="model_id"/>
+                            </tree>
+                        </field>
+                    </page>
+                    <page string="Logs">
+                        <field name="log_ids"
+                               attrs="{'readonly': [('id', '=', False)]}"
+                               context="{'default_namespace_id': active_id}"
+                        >
+                            <tree>
+                                <field name="create_date"/>
+                                <field name="create_uid"/>
+                                <!--TODO: maybe we need such a field?-->
+                                <!--<field name="request_method"/>-->
+                            </tree>
+                        </field>
+                    </page>
+                </notebook>
+              </sheet>
+            </form>
+        </field>
+    </record>
+
+
+    <record model="ir.ui.view" id="access_ir_exports_form_view">
+        <field name="name">openapi.access.exports.form</field>
+        <field name="model">ir.exports</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="export_fields"
+                   attrs="{'readonly': [('id', '=', False)]}"
+                />
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="access_form_view">
+        <field name="name">openapi.access.form</field>
+        <field name="model">openapi.access</field>
+        <field name="arch" type="xml">
+            <form string="Access">
+              <sheet>
+                <div class="oe_button_box" name="button_box">
+                    <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
+                        <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
+                    </button>
+                </div>
+                <div class="oe_title">
+                    <group>
+                        <field name="id" invisible="1"/>
+                        <field name="namespace_id"
+                               attrs="{'invisible': [('namespace_id', '!=', False)]}"
+                        />
+                        <field name="model" invisible="1"/>
+                        <field name="model_id"
+                               options="{'no_create': True, 'no_open': True}"
+                        />
+                        <field name="api_create"/>
+                        <field name="api_read"/>
+                        <field name="api_update"/>
+                        <field name="api_delete"/>
+                        <field name="api_public_methods"/>
+                        <field name="public_methods"/>
+                        <field name="private_methods"/>
+                        <!--TODO: namespace_id.name instead str(namespace_id)-->
+                        <field name="read_one_id"
```

Comment:
@yelizariev Why do we use M2O to ir.exports for the read_one_id and for read_many_id fields? I think we could use M2M to ir.model.fields (with model_id in domain) for this. Then the interface would be more clear.